Parsedown.php

```diff
@@ -75,6 +75,25 @@ function setUrlsLinked($urlsLinked)
 
     protected $urlsLinked = true;
 
+    function setSafeLinksEnabled($safeLinksEnabled)
+    {
+        $this->safeLinksEnabled = $safeLinksEnabled;
+
+        return $this;
+    }
+
+    protected $safeLinksEnabled = true;
+
+    protected $safeLinksWhitelist = array(
```

Comment:
Does this have to be an instance member? Seems like it can be moved to `filterUnsafeUrlInAttribute`